dist/index.html

```diff
@@ -109,16 +121,18 @@
 <body class="swagger-section">
 <div id='header'>
   <div class="swagger-ui-wrap">
-    <a id="logo" href="http://swagger.io">swagger</a>
-    <form id='api_selector'>
-      <div class='input'><input placeholder="http://example.com/api" id="input_baseUrl" name="baseUrl" type="text"/></div>
-      <div class='input'><input placeholder="api_key" id="input_apiKey" name="apiKey" type="text"/></div>
-      <div class='input'><a id="explore" href="#" data-sw-translate>Explore</a></div>
-    </form>
+    <div id="logo"></div>
   </div>
 </div>
 
 <div id="message-bar" class="swagger-ui-wrap" data-sw-translate>&nbsp;</div>
+<div class="swagger-ui-wrap">
+  <form id='api_selector'>
+    <!-- <div class='input'><input placeholder="http://example.com/api" id="input_baseUrl" name="baseUrl" type="text"/></div> -->
+    <div class='input'><input placeholder="Enter your apiKey here..." id="input_apiKey" name="apiKey" type="text"/></div>
```

Comment:
It looks like this will use `api_key` as the query parameter for the request. I couldn't immediately see any way to change what the was named. If that's the case, we need to change it to be `apiKey`.